README.md

```diff
@@ -181,3 +184,14 @@ Example:
 - Updates config bonfire file and add rbac component
 - Deploys rbac together with relationships application
   - Hardcoded image is used with grpc client for calling relationships
+
+## Validating FIPS
+Inventory API is configured to build with FIPS capable libraries and produce FIPS capaable binaries when running on FIPS enabled clusters.
+To validate the current running container is FIPS capable:
+```shell
+# exec or rsh into running pod
+# using fips-detect (https://github.com/acardace/fips-detect)
+fips-detect /usr/local/bin/inventory-api
+# using go tool
+go tool nm /usr/local/bin/inventory-api | grep FIPS
```

Comment:
Looks like a small copy-paste mishap here, should be `Relations API`

Reply:
And I believe `/usr/local/bin/kessel-relations`

Reply:
absolutely is haha, thanks!